Templates/JenkinsPipeline/Jenkinsfile

```diff
@@ -591,28 +608,32 @@ pipeline {
 							// Locate the UCD Component Version URL in the output log.
 							def ucdVersionUrl = searchLogOutput("version.url -> (.*)", readFile("${ucdBuzToolLog}"))
 							if (ucdVersionUrl){
-								println("${PipelineName}[INFO]: UCD Component Version URL: ${ucdVersionUrl}")
-								createSummary icon:"accept.svg", text: "${MsgHdr} <a href=\'$ucdVersionUrl\' target=\'_other\'>UCD Component Version (name: $ucdVersionName)</a>"
+								Msg = "Package succesfully created."
+								println("${PipelineName}[INFO]:")
+								addSummary  icon:"symbol-gift plugin-ionicons-api", text: "${MsgHdr} ${Msg}", style: "color: var(--success-color)"
+								addSummary  icon:"symbol-link-sharp plugin-ionicons-api", text: "${MsgHdr} UCD Component Version URL: ${ucdVersionUrl}", link:"${ucdVersionUrl}", target: "_blank"
 							} else {
 								Msg = "Fetching the UCD Component Version URL failed. Please check ${ucdBuzToolLogName}"
-								createSummary icon:"warning.svg", text: "${MsgHdr} ${Msg}"
+								addSummary icon:"symbol-warning-outline plugin-ionicons-api", text: "${MsgHdr} ${Msg}", style: "color: var(--warning-color)"
```

Comment:
Should the message also be printed to the console output, like on the other stages?